### ISSUE TITLE
picture.cc:448:62: error: 'mkdtemp' was not declared in this scope

### ISSUE THREAD

**Issue by mojca** [vectorgraphics/asymptote#212]
On Solaris 10 asymptote no longer compiles due to
```
picture.cc:448:62: error: 'mkdtemp' was not declared in this scope
```

I was curious since it used to compile last year and it seem that this has been added in commit fa3122974baecf2d7743e87705f3b12c913e560b.

Here's how it has been solved by Transmission:
https://trac.transmissionbt.com/changeset/13080

And an old thread:
http://lists.opencsw.org/pipermail/maintainers/2016-March/020282.html

**Comment by johncbowman** [comment]
Instead of implementing our own mkdtemp, another option is to require dvisvgm 2.10 or later, which includes this fix: https://github.com/mgieseki/dvisvgm/commit/2c8e03f5256518a3f7cb7fa6b275c33460d6e2e6
Then we can revert these commits:
24b51055076bd6340103cc0bed077932234f49f1
b9f5aa77eac95d201c88f515a79397e7dc95352c
fa3122974baecf2d7743e87705f3b12c913e560b

**Comment by kberry** [comment]
John, since you asked ... TL21 will have dvisvgm 2.11.1 (or later).

**Comment by johncbowman** [comment]
Fixed in d41e41b63bac1276aa6b5def965acc56188327d4
